tox.ini

```
@@ -1,15 +1,17 @@
 [tox]
 envlist =
-    py310-dj{main,41,40,32}-postgres
-    py39-dj{main,41,40,32}-postgres
-    py38-dj{main,41,40,32}-postgres
+    py311-dj{main,42,41}-postgres
+    py310-dj{main,42,41,40,32}-postgres
+    py39-dj{main,42,41,40,32}-postgres
+    py38-dj{main,42,41,40,32}-postgres
     py37-dj{32}-postgres
     linting
 
 [testenv]
 extras = testing
 deps =
     djmain: https://github.com/django/django/archive/main.tar.gz
+    dj42: Django>=4.2a1,<4.3
```

```suggestion
    dj42: Django>=4.2,<4.3
```
Use django 4.2 stable